ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-COSC-nonprod.json

```diff
@@ -72,5 +72,17 @@
        "CRUD": "CRU"
      }
    ]
+ },
+  {
+   "CaseTypeID": "CIVIL",
+   "CaseEventID": "NOTIFY_APPLICANT_SOLICITOR1_FOR_PAID_IN_FULL_COSC",
+   "AccessControl": [
```

Comment:
Should this event visible to caseworker? If it so then we need to add read permission for casewokers?

Reply:
updated